flink-core/pom.xml

```diff
@@ -33,6 +33,25 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config> <!--
+			required by JmxServer
+			-->--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED <!--
+			PluginConfigTest
+			-->--add-opens=java.base/java.util=ALL-UNNAMED <!--
+			ExceptionUtilsTest
+			-->--add-opens=java.base/java.lang=ALL-UNNAMED <!--
+			-->--add-opens=java.base/java.lang.invoke=ALL-UNNAMED <!--
```

Comment:
I didn't find any occurrences of `invoke` in `ExceptionUtilsTest` :thinking:

Reply:
I'll double-check where this comes from.

Reply:
hmm I can't reproduce the original problem. Will run Ci without this one and see what happens; could be a copy-paste error? (there's bound to be one somewhere)

Reply:
entry was removed.